js/Events/logInEvents.js

```diff
@@ -1,44 +1,47 @@
 const addEventsLogIn = function (){
-
     const submitFormEvent = function (event) {
     
-    event.stopPropagation();
-   event.preventDefault();
+        event.stopPropagation();
+        event.preventDefault();
 
-    const userNameInput = document.querySelector('#userNameInput');
-    const passwordInput = document.querySelector('#passwordInput');
+        const userNameInput = document.getElementById('userNameInput');
+        const passwordInput = document.getElementById('passwordInput');
+        const userNameValue = userNameInput.value;
+        const passwordValue = passwordInput.value;
 
-    const userNameValue = userNameInput.value;
-    const passwordValue = passwordInput.value;
+        if(checkCredentials(userNameValue, passwordValue)){
+            const app = document.getElementById('app');
+            app.innerHTML = EvaluationsPage();
+            addEventsEvaluations();
+        } else {
+            displayAlert(event.target, "Wrong username or password!");
+        }
 
-    if(checkCredentials(userNameValue, passwordValue)){
-        event.target.submit();
-    } else {
-        displayAlert(event.target, "Wrong username or password!");
-    }
-    
-        
-};
+    };
 
-const displayAlert = function(parent, message) {
-    var alertLabel = document.createElement('p');
-    alertLabel.className = "alert";
-    alertLabel.innerHTML = message;
-    parent.appendChild(alertLabel);
-};
+    const displayAlert = function(parent, message) {
+        var alert = document.querySelector('.alert');
 
-const checkCredentials = function(username, password){
-    
-    if(username ==="adrian" && password ==="frunza"){
-        return true;
-    } else {
-        return false;
-    }
-};
+        if(!alert){
```

Comment:
what happens if you already have an alert element inside this parent?

Reply:
i don't create one

Reply:
what if you have a different error to display?

Reply:
the function that creates the alert box does not continue the execution if an alert box is already present; this means that any message you pass will be ignored by the execution thread as the condition to add it inside the view will not be met.